scripts/upgrade

```diff
@@ -49,6 +49,10 @@ ynh_app_setting_delete --app=$app --key=email # also delete unnecessary duplicat
 email_fullname="$(ynh_user_get_info --username=$admin --key=fullname)"
 email="$(ynh_user_get_info --username=$admin --key=mail)"
 
+api_secret="$(ynh_string_random --length=32)"
```

Comment:
This will set new keys on every upgrade.
The `api_secret` is like a password, used to login, so if after every update this changes, this will break the users workflow.

Reply:
pushing the branch helpers 2.1 that adresses the issue on setting `api_secret` if not set.

Reply:
I'm not sure I understand this..

I understand that `ynh_app_setting_set_default --key=api_secret --value="$(ynh_string_random --length=32)"` will set `api_secret` if it doesn't exist yet, but won't the line `api_secret="$(ynh_string_random --length=32)"` overwrite this then?